src/panels/lovelace/cards/hui-glance-card.js

```diff
@@ -90,6 +96,11 @@ class HuiGlanceCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
   setConfig(config) {
     this._config = config;
     this.updateStyles({ '--glance-column-width': (config && config.column_width) || '20%' });
+
+    if (config && config.theming && typeof(config.theming) === 'string') {
+        this.classList.add(`theme-${config.theming || 'default'}`);
```

Comment:
`|| default` is never used as you already checking for truthiness with `if (config.theming)`